src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -36,11 +36,20 @@ public static void ChangeExtension(string path, string newExtension, string expe
         }
 
         [Fact]
-        public static void GetDirectoryName_EmptyThrows()
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]
+        public static void GetDirectoryName_EmptyThrows_Desktop()
         {
             AssertExtensions.Throws<ArgumentException>("path", null, () => Path.GetDirectoryName(string.Empty));
         }
 
+        [Fact]
+        [ActiveIssue(27269)]
```

Comment:
#27269 seems to be they issue for this PR. Wrong number?

Reply:
That should be right. This is disabling the test. I'll change the issue to re-enable after corelib is consumed.